tests/test_binary_pointwise_ops.py

```diff
@@ -1207,3 +1207,19 @@ def test_accuracy_allclose(shape, dtype, equal_nan, gen_nan):
     ref_out = torch.allclose(ref_inp1, ref_inp2, rtol, atol, equal_nan=equal_nan)
 
     assert res_out == ref_out
+
+
+@pytest.mark.logical_or
+@pytest.mark.parametrize("shape", POINTWISE_SHAPES)
+@pytest.mark.parametrize("dtype", ALL_FLOAT_DTYPES)
```

Comment:
Add tests for BOOL_TYPES and INT_DTYPES.

Reply:
Fixed, thx